lib/strategies/default.js

```diff
@@ -1,9 +1,24 @@
 'use strict';
 
+var Promise = require('rsvp').Promise;
+
 module.exports = {
   init: function() {},
 
   processString: function(ctx, contents, relativePath) {
-    return ctx.processString(contents, relativePath);
+    var string =  Promise.resolve(ctx.processString(contents, relativePath));
```

Comment:
should likely take the constructor form, so if processString itself throws this function is guarenteed to return a promise